assets/dev/scss/frontend/widgets/divider.scss

```diff
@@ -137,11 +137,28 @@
 	direction: rtl;
 }
 
-.e-container {
+.elementor {
```

Comment:
I wonder if this parent `.elementor` container is necessary.
Can you check if it is necessary - maybe it is to get the specificity that you are needing. but maybe it;s also just superflous and not needed, in which case you can remove it.

Reply:
@StuartElementor I have to increase the specificy for BC reasons.